index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+  <meta charset="utf-8">
+
+  <title>Audioworklet with polyfill</title>
+  <script src="audioworklet.js"></script>
+  <script src="console.js"></script>
+
+</head>
+
+<body>
+  <pre class="console">
+
+    </pre>
+  <script>
+    // The reason we use ConsoleJS is so that its easier
+    // to see console logs in all devices, including phones.
+    ConsoleJS.init({
+      selector: "pre.console"
+    });
+
+    // audioworker.js should also reside at root
+    const audioContext = new AudioContext();
+
+    // -- buflenSPN defines ScriptProcessorNode buffer length in samples
+    // -- default is 512. use larger values if there are audible glitches
+    AWPF.polyfill(audioContext, { buflenSPN: 512, buflenAWP: 512 }).then(() => {
+
+      console.log("Loading audioprocessor");
+      // Initialize module and create a node for processing.
+      return audioContext.audioWorklet.addModule("audioprocessor.js");
+    }).then(() => {
+      console.log("Getting stream");
+      // Get stream and create audio graph source
+      return navigator.mediaDevices.getUserMedia({ audio: {
+        noiseSuppression: false,
+        echoCancellation: false,
+        autoGainControl: false
+      } });
+    }).then((stream) => {
+      console.log("Creating audio graph");
+      const mic = audioContext.createMediaStreamSource(stream);
+      let pn = new AudioWorkletNode(audioContext, "audioprocessor", { numberOfInputs: 1, numberOfOutputs: 1, outputChannelCount: [2] });
```

Comment:
Both `numberOfInputs` and `outputChannelCount` are necessary options for the polyfill to work correctly.
![image](https://user-images.githubusercontent.com/16245985/81136976-86e09700-8f76-11ea-874f-471df8a14547.png)
![image](https://user-images.githubusercontent.com/16245985/81137043-cad39c00-8f76-11ea-9ee7-5ae7f757f9bc.png)